Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -403,7 +403,21 @@ import Foundation
     func fetchAPITiming(path: String, connectionStartTime: Int?, requestStartTime: Int?, startTime: Int, endTime: Int) {
         let cleanedPath = path.replacingOccurrences(of: "/merchants/([A-Za-z0-9]+)/client_api", with: "", options: .regularExpression)
 
-        if cleanedPath != "/v1/tracking/batch/events" {
+        if cleanedPath.contains("three_d_secure") {
+            let threeDSecurePath = cleanedPath.replacingOccurrences(
+                of: "payment_methods/.*/three_d_secure",
+                with: "payment_methods/three_d_secure",
+                options: .regularExpression
+            )
```

Comment:
Do we need the `if cleanedPath.contains("three_d_secure")` check at all? I would assume that the regex on lines 407-411 would be fine to run against _all_ paths, and it would only modify the ones that matched. This way we only need to call `analyticsService?.sendAnalyticsEvent` in 1 place

Reply:
🤦‍♂️ you're totally right, updated here: https://github.com/braintree/braintree_ios/pull/1375/commits/00fade197d79f4004aafd7f845a6a5c9bd2a3354